TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3185,3 +3185,14 @@ def elevation_to_meters(shape, props, fid, zoom):
 
     props['elevation'] = _to_float_meters(elevation)
     return shape, props, fid
+
+
+def add_is_bicycle_route(shape, props, fid, zoom):
+    """
+    If the props contain a bicycle_network tag or cycleway, it should
+    have an is_bicycle_route boolean.
+    """
+    props.pop('is_bicycle_route', None)
+    if 'bicycle_network' in props or 'cycleway' in props:
```

Comment:
> Ah, dropping no is covered here: https://github.com/mapzen/vector-datasource/pull/780/files.

Yup, I mentioned it in the pr description.

> Why we have 4 properties for all these instead of the 1 is a little weird, but I think just inherited from OSM?

Yea, I just included it because I saw it in the [osm wiki](http://wiki.openstreetmap.org/wiki/Bicycle). It does have some [decent coverage](http://taginfo.openstreetmap.org/keys/cycleway:left#values), although about an order of magnitude less than just [cycleway](http://taginfo.openstreetmap.org/keys/cycleway#values).

I think what you're saying makes sense though, if we're including them we might as well have consistent behavior for each. I'll go ahead and do that.